### ISSUE TITLE
ChatSecure doesn't work in the background

### ISSUE THREAD

**Issue by OmlineEditor** [ChatSecure/ChatSecure-iOS#1229]
I open the app. my friends see me and they can write me messages and I get them.
I minimize the app (don't close it), wait 5 minutes.
my friends write to me, but I don't get a message.
I've been waiting for hours, but the message doesn't come. My friends see me as offline.
I launch the app a few hours later and I get a message. 

iPhone 10
iOS 14.3
ChatSecure 5.0.2

XMPP server: Prosody 0.11.7
included modules: XEP-0352,  XEP-0357,  XEP-0198

Tell me if you need more data

**Comment by bsima** [comment]
I'm seeing the same thing. What can we do to help implement/fix background chat notifications?

**Comment by licaon-kter** [comment]
Switch to Android :)

Or try Siskin iOS (or its fork Snikket iOS) or Monal iOS?

**Comment by chrisballinger** [comment]
Yeah, unfortunately I'd recommend checking out alternatives these days
